### PR TITLE
Delay comment notification with Oban and allow for comment editing by author

### PR DESCRIPTION
**Pull request by akoutmos**

### CODE REVIEW

File: lib/changelog/prom_ex.ex [hunk 1]
```diff
@@ -8,7 +8,9 @@ defmodule Changelog.PromEx do
       {PromEx.Plugins.Application, otp_app: :changelog},
       PromEx.Plugins.Beam,
       {PromEx.Plugins.Phoenix, router: ChangelogWeb.Router},
-      {PromEx.Plugins.Ecto, otp_app: :changelog, repos: [Changelog.Repo]}
+      {PromEx.Plugins.Ecto, otp_app: :changelog, repos: [Changelog.Repo]},
+      PromEx.Plugins.Oban
```
Comment (jerodsanto) [inline lib/changelog/prom_ex.ex:12]: Does this mean PromEx will run its stuff via Oban automatically now?

File: lib/changelog_web/controllers/news_item_comment_controller.ex [hunk 2]
```diff
@@ -56,6 +59,41 @@ defmodule ChangelogWeb.NewsItemCommentController do
     html(conn, NewsItemCommentView.transformed_content(markdown))
   end
 
+  def update(conn, %{"id" => id, "news_item_comment" => %{"content" => updated_content}}) do
+    with id <- Changelog.Hashid.decode(id),
```
Comment (jerodsanto) [inline lib/changelog_web/controllers/news_item_comment_controller.ex:63]: deft use of `with` here 👍

**Review by jerodsanto — CHANGES_REQUESTED** [review]
Looking pretty good!

File: lib/changelog/prom_ex.ex (hunk 1 above)
Comment (akoutmos) [reply lib/changelog/prom_ex.ex:12]: That's correct! PromEx will automatically pick up on any Oban events that are emitted and those metrics will automagically find their way to the included Grafana Oban dashboard :)

File: lib/changelog_web/controllers/news_item_comment_controller.ex (hunk 2 above)
Comment (akoutmos) [reply lib/changelog_web/controllers/news_item_comment_controller.ex:63]: I thought so too haha

File: lib/changelog_web/helpers/shared_helpers.ex [hunk 3]
```diff
@@ -194,4 +194,17 @@ defmodule ChangelogWeb.Helpers.SharedHelpers do
   def word_count(text) when is_binary(text) do
     text |> md_to_text() |> String.split() |> length()
   end
+
+  def can_edit_comment?(
```
[the remark this answers is not in the document]
Comment (akoutmos) [reply lib/changelog_web/helpers/shared_helpers.ex]: Good point! Addressed in the latest commit.

**Pull request merged**